source/timerFormatter.mc

```diff
@@ -1,5 +1,16 @@
 
 function timerFormat(seconds as Number) {
-    // Formatting code here
-	return "Your Answer";
+    var hour = seconds / 3600; // decimal
+    var min = (seconds % 3600) / 60; // (remainder of seconds/3600) / 60 // decimal
+    var sec = seconds % 60;
+
+    var retval = "";
+    if (hour >= 1) { // are we printing it at all?
+        retval += (hour >=10) ? hour.format("%02i") : hour.format("%01i");
```

Comment:
Awesome work, good comments and everything!

Not a big deal at all, one typo though, make sure there is equal spacing for "hour <= 10". I know it's hard because the auto formatting for MonkeyC is no good. But I thought I would point this out so we could go through the normal [Open PR] -> [Fix PR] flow, just to get you more familiar with it.

Go back to your branch and make the change, and just push the commit to the same branch and it will show me it's updated.

After that I will approve it, and merge the PR.

Really good stuff though! I'll start working on getting the next lesson setup so we can discuss it Friday.

Reply:
Perfect! Merged